pynetdicom/apps/tests/test_echoscu.py

```diff
@@ -108,7 +108,7 @@ def test_no_peer(self, capfd):
 
         out, err = capfd.readouterr()
         assert "Association request failed: unable to connect to remote" in err
-        assert "TCP Initialisation Error: Connection refused" in err
```

Comment:
Logging the particular `connect` exception adds an errno to this log which can vary across platforms

Reply:
Ha, and the message can vary as well..I think we should do:

`TCP Initialisation Error:\n{exc}`

and check for `TCP Initialisation error` in tests